proposals/p3848.md

```diff
@@ -0,0 +1,587 @@
+# Lambdas
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/3848)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Syntax Overview](#syntax-overview)
+    -   [Succinctly](#succinctly)
+    -   [Alternative Considered: Terse vs Elaborated](#alternative-considered-terse-vs-elaborated)
+-   [Introducer](#introducer)
+    -   [Alternative Considered: Sigil](#alternative-considered-sigil)
+-   [Positional Parameters](#positional-parameters)
+    -   [Positional Parameter Restrictions](#positional-parameter-restrictions)
+        -   [Alternative Considered: Additional Restriction](#alternative-considered-additional-restriction)
+-   [Function Captures](#function-captures)
+    -   [Capture Modes](#capture-modes)
+        -   [Future Work: Reference Captures](#future-work-reference-captures)
+    -   [Default Capture Mode](#default-capture-mode)
+        -   [Alternative Considered: Middle vs Beginning](#alternative-considered-middle-vs-beginning)
+-   [Function Fields](#function-fields)
+-   [Copy Semantics](#copy-semantics)
+-   [Self and Recursion](#self-and-recursion)
+    -   [Alternative Considered: Recursive Self](#alternative-considered-recursive-self)
+-   [Rationale](#rationale)
+
+<!-- tocstop -->
+
+## Abstract
```

Comment:
It would be good to add a background section that provides reference links for Lambdas in other languages -- at least C++, Swift, and Rust.